Marlin/example_configurations/delta/FLSUN/auto_calibrate/Configuration.h

```diff
@@ -446,53 +446,56 @@
   // and processor overload (too many expensive sqrt calls).
   #define DELTA_SEGMENTS_PER_SECOND 160
 
+  // NOTE NB all values for DELTA_* values MUST be floating point, so always have a decimal point in them
+
   // Center-to-center distance of the holes in the diagonal push rods.
   #define DELTA_DIAGONAL_ROD 218.0 // mm
-
+/*
```

Comment:
I left it if someone wants to use all those offsets to 'calculate' the delta_radius; they can uncomment it.